### ISSUE TITLE
Move Notificatio's code to Django instead of JS

### ISSUE THREAD

**Issue by JonanOribe** [CiviWiki/OpenCiviWiki#1490]
### Idea summary

Looking at project\core\templates\static\js\notifications.js, I saw that the logic behind notification's component is running over JS. Maybe will be better to move this to Django/Python syntax to unify the platform.

### Further details

Will be easier to rewrite this code as DJANGO {% if %} {% else%}, that use JS on one side of the project and Python on another

![image](https://user-images.githubusercontent.com/11914201/215096420-093a45eb-ff5f-4fda-b1aa-fe6d667b0f28.png)

I could take this task if necessary

**Comment by JonanOribe** [comment]
@brylie looking at the admin panel, I found that we have two diferent types of notifications. Could be interesting change the name of one of them to avoid misunderstandings

![image](https://user-images.githubusercontent.com/11914201/215435541-8af84221-24cd-475b-8e6a-af0a2d4df736.png)

One is like a notification like "something happens" (new follower) and the other one is a notification on detail. What you think about this?

**Comment by brylie** [comment]
Yeah, we should clarify the difference between those notifications. Can you find the places in the code where both types of notifications are defined?

**Comment by Lvcky-gg** [comment]
I can take this

**Comment by sambarvadai** [comment]
Hi @brylie. If there's any help needed, can I give this a go?
